operations/observability/Observability.md

```diff
@@ -0,0 +1,365 @@
+# Install and Upgrade Observability Framework
```

Comment:
I just noticed that this file is in a new observability subdirectory which contains no other files. Are you anticipating there being multiple documents relating to observability? If not, would it make more sense to put this file directly in the iuf directory instead?

Reply:
There will be more files in the future as we have more observability work till the end of this year.